### PR TITLE
search functionality

### LINKED ISSUE #31: search in health book

### PR DESCRIPTION
**Pull request by isabellapedraza**
implemented search for contact and medication. Did not implement search for entry yet, as I believe it might be easier to implement once entries and medications/contacts are synced.

### CODE REVIEW

File: client/components/Medication/MedicationPage.vue [hunk 1]
```diff
@@ -39,3 +61,14 @@ export default {
   methods: {},
 };
 </script>
+
+<style scoped>
+
+/* h2 {
```
Comment (jrdegreeff) [inline client/components/Medication/MedicationPage.vue:67]: remove comment before merging

**Review by jrdegreeff — DISMISSED** [review]
I'm not the biggest fan of the duplication, but this seems fine overall to me. Are you planning to merge this PR as is or wait to complete the entire issue? I'm fine either way, but if the former you should detach the issue so that it isn't closed with the partial completion.

File: client/components/Medical-Contact/ContactPage.vue [hunk 2]
```diff
@@ -13,9 +13,17 @@
     <section>
       <header>
         <h2>Current Contacts</h2>
+        <div class="search-wrapper">
+          <label>Search by contact last name:</label>
+          <input
```
Comment (wli-linda) [inline client/components/Medical-Contact/ContactPage.vue]: I wonder if it'd be worth the effort to make contacts searchable by any part of the name, not just last name?

**Comment by isabellapedraza** [comment]
@jrdegreeff I think I'll make a separate issue for implementing search on entries – and keep the current issue attached to this branch so we can merge to main

**Comment by jrdegreeff** [comment]
> @jrdegreeff I think I'll make a separate issue for implementing search on entries – and keep the current issue attached to this branch so we can merge to main

sounds good to me

**Review by jrdegreeff — APPROVED** [review]

**Pull request merged**